app/poros/github_search.rb

```diff
@@ -5,18 +5,22 @@ def initialize(token)
     @token = token
   end
 
+  def conn(path)
```

Comment:
This is awesome and exactly what Brian was talking about. Thanks for making these changes!